break_camel_case.py

```diff
@@ -0,0 +1,7 @@
+import re
+def break_camel_case(s):
+    str = ''
```

Comment:
`str` is the built in python class that creats a string ie. str(123) --> "123" so you should not override it in your function. Alos variable should not be named after their python type, but their contents, so result, characters, or words are better variable names.